src/components/common/Footer.tsx

```diff
@@ -0,0 +1,95 @@
+import { FC } from 'react';
+import Link from 'next/link';
+
+import { Logo } from './Logo';
+import { Icon } from './Icon';
+import { isExternalUrl } from '../../utils/helpers';
+import config from '../../config';
+import * as routes from '../../routes';
+
+const content = {
+  note: (
+    <p className="mt-2">
+      Made with 💜 by{' '}
+      <Link href={config.raydiantUrl}>
+        <a
+          className="hover:text-slate-700 dark:hover:text-slate-300"
+          target="_blank"
+          rel="noreferrer"
+        >
+          Raydiant
+        </a>
+      </Link>
+    </p>
+  ),
+  menus: [
+    {
+      // TODO: Populate
+      title: 'Docs',
+      elements: [{ label: 'Get Started', url: routes.getStarted() }],
+    },
+    {
+      // TODO: Populate
+      title: 'Examples',
```

Comment:
Should this link to our sample apps?
- NextJS / WholeFoods 
- Vanilla / Starbucks
...

Reply:
Yeah probably. Let me think about what to put in the footer and get back to you

Reply:
Here's what I'm thinking for the footer. Can be a separate PR if you want.

<img width="786" alt="image" src="https://user-images.githubusercontent.com/1163252/174411729-b7d0f938-ec9b-4521-aa6d-e54a65f257cc.png">

https://www.notion.so/raydiant/Developer-Docs-Content-56130d18cbd740508ffc0f42b62089de

Reply:
Thanks 🙏 
Card to track that here: https://trello.com/c/6wiUCVvl/4345-developer-docs-site-populate-footer-with-content